### DISCUSSION TITLE
执行scripts/run.sh时错误 提示no module name requests 错误文件为generateWSALinks.py

### DISCUSSION THREAD

**Discussion by zhq123333** [LSPosed/MagiskOnWSALocal#448] [Q&A]
执行scripts/run.sh时错误 提示no module name requests 错误文件为generateWSALinks.py

**Comment by Howard20181** [comment]
请给更多信息
你的系统可能太旧了

**Reply by zhq123333** [reply]
系统是win11家庭版 22621.1413 
我使用之前的版本:
[87ec298](https://github.com/LSPosed/MagiskOnWSALocal/commit/87ec298263f5fe1d2eb8ca498ecaf78c544a081e)
可以运行成功并生成output 但是最新的版本会提示没有找到requests模组的错误
错误讯息：/scripts/generateWSALinks.py", line 31, in <module>
    from requests import Session
ModuleNotFoundError: No module named 'requests'

**Reply by Howard20181** [reply]
问的是Linux系统信息
这是Python模组，可以自动安装的

**Reply by zhq123333** [reply]
linux子系统版本是5.10.16

**Reply by Howard20181** [reply]
发行版版本

**Reply by zhq123333** [reply]
刚才跑了下linux update就正常了，的确是版本太旧，谢谢你的帮助
